PROTOCOL.md

```diff
@@ -1835,3 +1836,27 @@ uppercase and lowercase as part of percent-encoding. Thus, we require a stricter
 3. Always [percent-encode](https://datatracker.ietf.org/doc/html/rfc3986#section-2) reserved octets
 4. Never percent-encode non-reserved octets
 5. A percent-encoded octet consists of three characters: `%` followed by its 2-digit hexadecimal value in uppercase letters, e.g. `>` encodes to `%3E`
+
+## Delta Data Type to Parquet Type Mappings
+Below table captures how each Delta data type is stored physically in Parquet files. Parquet files are used for storing the table data or metadata ([checkpoints](#checkpoints)). Parquet has a limited number of [physical types](https://parquet.apache.org/docs/file-format/types/). Parquet [logical types](https://github.com/apache/parquet-format/blob/master/LogicalTypes.md) are used to extend the types by specifying how the physical types should be interpreted.
+
+For some of the Delta data types, there are multiple ways store the values physically in Parquet file. For example, `timestamp` can be stored either as `int96` or `int64`. The exact physical type depends on the engine that is writing the Parquet file and/or engine specific configuration options. For a Delta lake table reader, it is recommended that the Parquet file reader support at least the Parquet physical and logical types mentioned in the below table.
+
+Delta Type Name | Parquet Physical Type | Parquet Logical Type 
+-|-|-
+boolean| `boolean` |
+byte| `int32` | `INT(bitwidth = 8, signed = true)`
+short| `int32` | `INT(bitwidth = 16, signed = true)`
+int| `int32` | `INT(bitwidth = 32, signed = true)`
+long| `int64` | `INT(bitwidth = 64, signed = true)`
+date| `int32` | `DATE`
+timestamp| `int96` or `int64` | `TIMESTAMP(isAdjustedToUTC = true, units = microseconds)`
```

Comment:
Should we recommend int64 as preferred? int96 is deprecated: https://github.com/apache/parquet-format/pull/86

Reply:
Got. Lets merge this PR and then we can look at what should be the preferred one, because it affects the existing readers and need to see if all existing readers have support for int64 as timestamp.

Reply:
second this. I think we should at least mandate all timestamp cols written after some date to be int64.

Reply:
@lzlfred are you aware of any reader that doesn't support int64 yet? 

Context: In Kernel the [`ParquetHandler`](https://github.com/delta-io/delta/blob/master/kernel/kernel-api/src/main/java/io/delta/kernel/client/ParquetHandler.java#L94) for writes takes `ColumnarBatch` which has schema as `StructType`. There is no clear way to communicate with the current API whether to write the timestamp column as INT96 or INT64. There are no configuration options like we have in Delta-Spark. We need 
 - update the API to indicate what type of physical Parquet format we want for a given column
    - could be in the metadata of `StructField`. 
 - make the writer always write as INT64 (checking if any Delta clients have problem with or not support it yet)